CRM/SMS/Form/Upload.php

```diff
@@ -90,6 +93,16 @@ public function setDefaultValues() {
 
     $defaults['upload_type'] = 1;
 
+    $showHide = new CRM_Core_ShowHideBlocks();
+    if ($skipTextFile) {
```

Comment:
This line wouldn't work anymore if you remove those bits.
What I meant was none of the show/hide/skiptext seemed to be doing anything different than before.